include/pmacc/assert.hpp

```diff
@@ -24,35 +24,65 @@
 
 #include "pmacc/debug/abortWithError.hpp"
 
+#include <cassert>
+
 #ifdef NDEBUG
 // debug mode is disabled
 
 /* `(void)0` force a semicolon after the macro function */
 #    define PMACC_ASSERT(expr) ((void) 0)
 
+/* `(void)0` force a semicolon after the macro function */
+#    define PMACC_DEVICE_ASSERT(expr) ((void) 0)
+
 /* `(void)0` force a semicolon after the macro function */
 #    define PMACC_ASSERT_MSG(expr, msg) ((void) 0)
 
+// debug mode is disabled
+/* `(void)0` force a semicolon after the macro function */
+#    define PMACC_DEVICE_ASSERT_MSG (expr, ...)((void) 0)
```

Comment:
I am wondering that the CI passed because I would say there is a space missing `(expr, ...)((void) 0)` should be `(expr, ...) ((void) 0)`. Do you use `clang-format-11`?

Reply:
after `PMACC_DEVICE_ASSERT_MSG` there is a space to much

Reply:
For an unknown reason, this part is not wrong formatting. Never the less could you please apply the changes by hand to be consistent with the code around.